apps/studio/src/lib/db/clients/bedrock.ts

```diff
@@ -0,0 +1,351 @@
+  async listTableTriggers(
+    table: string,
+    _schema?: string
+  ): Promise<TableTrigger[]> {
+    const sql = `
+      SELECT name, sql
+      FROM sqlite_master
+      WHERE type = 'trigger'
+        AND tbl_name = '${table}'
+    `;
+
+    const { rows } = await this.driverExecuteSingle(sql);
+    return rows as TableTrigger[];
+  }
+
+  // Override listTableIndexes to use SQLite PRAGMA instead of information_schema
+  async listTableIndexes(
+    table: string,
+    _schema?: string
+  ): Promise<TableIndex[]> {
+    const sql = `PRAGMA index_list('${SD.escapeString(table)}')`;
+
+    const { rows } = await this.driverExecuteSingle(sql, {
+      overrideReadonly: true,
+    });
+
+    if (!rows || rows.length === 0) {
+      return [];
+    }
+
+    const allSQL = rows
+      .map((row) => `PRAGMA index_xinfo('${SD.escapeString(row.name)}')`)
+      .join(";");
+    const { rows: infoRows } = (await this.driverExecuteSingle(allSQL, {
+      overrideReadonly: true,
+    })) as any;
+
+    // Handle case where driverExecuteMultiple returns null or undefined
+    const safeInfos = infoRows || [];
+
+    const indexColumns: IndexColumn[][] = safeInfos.map((result) => {
+      return (result?.rows || [])
+        .filter((r) => !!r.name)
+        .map((r) => ({ name: r.name, order: r.desc ? "DESC" : "ASC" }));
+    });
+
+    return rows.map((row, idx) => ({
+      id: `${table}.${row.name}`,
+      table,
+      schema: "",
+      name: row.name,
+      columns: indexColumns[idx] || [],
+      unique: !!row.unique,
+      primary: row.origin === "pk",
+    }));
+  }
+  listSchemas(_filter?: SchemaFilterOptions): Promise<string[]> {
+    return Promise.resolve([]); // DOES NOT SUPPORT IT
+  }
+
+  getTableReferences(_table: string, _schema?: string): Promise<string[]> {
+    return Promise.resolve([]); // TODO: not implemented yet
+  }
+
+  // Override getTableKeys to use SQLite PRAGMA instead of information_schema
+  async getTableKeys(
+    table: string,
+    _schema?: string
+  ): Promise<SharedTableKey[]> {
+    const sql = `pragma foreign_key_list('${SD.escapeString(table)}')`;
+    const { rows } = await this.driverExecuteSingle(sql, {
+      overrideReadonly: true,
+    });
+    return rows.map((row) => ({
+      constraintName: row.id,
+      constraintType: "FOREIGN",
+      toTable: row.table,
+      toSchema: "",
+      fromSchema: "",
+      fromTable: table,
+      fromColumn: row.from,
+      toColumn: row.to,
+      onUpdate: row.on_update,
+      onDelete: row.on_delete,
+      isComposite: false, // SQLite foreign keys are typically single column
+    }));
+  }
+
+  // We also need to override parseTableColumn - let's use a simplified version
+  parseTableColumn(row: any): any {
+    return {
+      name: row.name,
+      type: row.type,
+      nullable: Number(row.notnull || 0) === 0,
+      defaultValue: row.dflt_value === "NULL" ? null : row.dflt_value,
+    };
+  }
```

Comment:
This should just return a BksField. That is used for our transcoders system, the simplest form would just be
```suggestion
  // We also need to override parseTableColumn - let's use a simplified version
  parseTableColumn(row: any): any {
    return {
      name: row.name,
      bksType: 'UNKNOWN'
    };
  }
```

The other information you put here is actually on the ExtendedTableColumn struct, not this one